samples/subsys/input/input_dump/sample.yaml

```diff
@@ -6,3 +6,11 @@ tests:
     build_only: true
     integration_platforms:
       - native_posix
+  sample.input.input_dump_shell:
+    tags: input
+    build_only: true
```

Comment:
could this be run and output verified?

Reply:
ah wait, it's a shell, so no. Maybe with the new pytest module, but not now

Reply:
Yeah not a bad idea, could add an overlay for native posix, simulate a click and check the console output, will look into it.

Reply:
_not a blocker_

Reply:
@fabiobaltieri You may be able to find some inspiration in the CAN shell module tests (https://github.com/zephyrproject-rtos/zephyr/tree/main/tests/drivers/can/shell). It instantiates a fake CAN driver and uses a dummy shell for interacting with it.

Reply:
Thanks @henrikbrixandersen I think I'll take that way.